Sources/RSDKUtils/EnvironmentInformation.swift

```diff
@@ -3,14 +3,11 @@ import Foundation
 @objc public class EnvironmentInformation: NSObject {
     let bundle: BundleProtocol
 
-    var appId: String {
-        return bundle.value(for: "RASApplicationIdentifier" as String) ?? bundle.valueNotFound
-    }
```

Comment:
Why remove appId?

Reply:
Because we don't use it anymore? I'm not sure, that was my guess. I can bring it back

Reply:
`appId` is internal and not used in the source code right?
Fine for me to remove it if it is not used.